modules/ggl-http/src/aws_sigv4.h

```diff
@@ -0,0 +1,62 @@
+// aws-greengrass-lite - AWS IoT Greengrass runtime for constrained devices
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef GGHTTPLIB_AWS_SIGV4_H
+#define GGHTTPLIB_AWS_SIGV4_H
+
+#include "ggl/http.h"
+#include <ggl/buffer.h>
+#include <ggl/error.h>
+#include <ggl/vector.h>
+
+#define ZERO_PAYLOAD_SHA \
```

Comment:
Could you put a comment on how to get this value? I.e. sha-256 hash of ""

Reply:
Yup.